File: Pregunta4_FuncionesIntegracion/Pregunta4.cs
using System;

class Pregunta4 {
    static string mostrarMenú() {
        Console.WriteLine("1) Cuadrado");
        Console.WriteLine("2) Rectángulo");
        Console.WriteLine("3) Círculo");
        Console.WriteLine("4) Salir");
        return Console.ReadLine();
    }

    static void Main() {
        while (true) {
            string op = mostrarMenú();
            if (op == "4") break;

            if (op == "1") {
                double lado = double.Parse(Console.ReadLine());
                Console.WriteLine($"{lado*lado:F2}");
                Console.WriteLine($"{4*lado:F2}");
            }
            else if (op == "2") {
                double b = double.Parse(Console.ReadLine());
                double h = double.Parse(Console.ReadLine());
                Console.WriteLine($"{b*h:F2}");
                Console.WriteLine($"{2*(b+h):F2}");
            }
            else if (op == "3") {
                double r = double.Parse(Console.ReadLine());
                Console.WriteLine($"{Math.PI*r*r:F2}");
                Console.WriteLine($"{2*Math.PI*r:F2}");
            }
        }
    }
}

File: Pregunta4_FuncionesIntegracion/pregunta4.py
import math

def mostrarMenu():
    print("1) Cuadrado")
    print("2) Rectángulo")
    print("3) Círculo")
    print("4) Salir")
    return input()

while True:
    op = mostrarMenu()
    if op == "4":
        break
    if op == "1":
        lado = float(input())
        print(f"{lado*lado:.2f}")
        print(f"{4*lado:.2f}")
    elif op == "2":
        b = float(input())
        h = float(input())
        print(f"{b*h:.2f}")
        print(f"{2*(b+h):.2f}")
    elif op == "3":
        r = float(input())
        print(f"{math.pi*r*r:.2f}")
        print(f"{2*math.pi*r:.2f}")
